config/config.go

```diff
@@ -108,11 +109,17 @@ func NewFromReader(reader io.Reader) (S3Cli, error) {
 		return S3Cli{}, fmt.Errorf("Invalid credentials_source: %s", c.CredentialsSource)
 	}
 
+	// For Alibaba Cloud, host is required.
```

Comment:
I don't understand what this comment is trying to explain, can you elaborate? Or maybe remove the comment?

Reply:
I want to give a tip that alicloud will not provide default region and the region comes from host, and host is required.